docs/details/shards/GFX/Drawable.md

```diff
@@ -0,0 +1,47 @@
+The drawable shard is used to define an instance of a drawn object, it usually has a transform (in world space) and a mesh to draw.
+
+--8<-- "includes/drawable_refactor.md"
+
+## Params
+
+The `Params` parameter contains a table of values that will be passed to the shader when this drawable is rendered.
+
+!!! info Defining shader parameters
+    See [GFX.Feature](../Feature) about where shader parameters are defined and used
+
+## Constant/Dynamic
+
+Shader parameters and the drawable transform can be set as either constant parameters or dynamic ones.
+
+### Constant parameters
+
+Constant parameters are passed through the input table. They are read and stay the same until this function is called again.
+
+```clojure
+{:Mesh .mesh :Transform .transform} (GFX.Drawable) >= .my-drawable
+```
+
+### Dynamic parameters
+
+Dynamic parameters are passed as parameters to the shard, the same fields can be set, except for the mesh. They are referenced and when their value changes will reflect this change in the rendered result.
```

Comment:
```suggestion
Dynamic parameters are passed in as parameters to the shard. The same fields can be set elsewhere, with the exception of the mesh. When they are referenced, changes in their value will be reflected in the rendered result.
```

Might need to reword this as the definition is a little unclear.

Reply:
Reworded